src/test/java/org/mockitousage/stubbing/StubbingWithAdditionalAnswersTest.java

```diff
@@ -86,11 +100,7 @@ public void will_execute_a_void_based_on_strongly_typed_one_parameter_function()
         final IMethods target = mock(IMethods.class);
 
         given(iMethods.simpleMethod(anyString()))
-                .will(answerVoid(new VoidAnswer1<String>() {
-                    public void answer(String s) {
-                        target.simpleMethod(s);
-                    }
-                }));
+                .will(answerVoid((VoidAnswer1<String>) target::simpleMethod));
```

Comment:
Please revert these changes. While we build locally on Java 8, we target Java 6 and I would like to make sure it keeps compiling on these older jdks. Our Travis build sadly does not test on jdk 6 any longer as it was deprecated and broken in their containers :sob: